### DISCUSSION TITLE
Laggy playback while streaming from internet

### DISCUSSION THREAD

**Discussion by AestraDev** [smplayer-dev/smplayer#546] [Q&A]
OS : windows 7 professional sp1 ( 32 bit )
Ram: 2gb
Processor: intel core 2 duo 2.53Gz

Greetings,
I have recently downgraded from windows 8.1 to windows 7 because of some reasons. I found that sm player was working like a miracle  for me in windows 8.1 but since I downgraded to windows 7 it gives a laggy playback I have tried several settings in preferences but none of them really work so well. Another problem I faced is that I can run videos from YouTube it gave a error something like "api_l1..... was missing from your computer try reinstalling the program" I I using same hardware I was using in windows 8.1 but it was working perfectly in that, I would appreciate a quick response to this problem
Thank you

**Comment by smplayer-dev** [comment]
It would be helpful if you could provide the full error message.
